lib/ui/screens/contact_detail_screen.dart

```diff
@@ -1,20 +1,120 @@
 
 import 'package:flutter/material.dart';
 import 'package:flutter_contact_list/mockdata/mock_contact.dart';
+import 'package:flutter_contact_list/pic/assets.dart';
 
 class ContactDetail extends StatelessWidget {
     final Contact contact; 
+
+    final fields = ["First name", "Last name", "Address", "Gender", "Email"]; 
+    final divider = Divider(
+        color: Colors.teal[900], 
+        thickness: 3,
+        indent: 200,
+        endIndent: 200,
+    ); 
+
+
     ContactDetail({Key? key, required this.contact}) : super(key: key); 
+
+    List<Widget> appBarPic(String contactName){
```

Comment:
Should create a dedicated StatelessWidget named ContactHeader to display Contact header. Put that widget in /ui/screens/contact_details/widgets/contact_header.dart